src/screens/exercise-screen/breathing-animation.tsx

```diff
@@ -15,8 +16,11 @@ interface Props {
   color?: string;
 }
 
-export const BreathingAnimation: FC<Props> = ({ animationValue, color = colors.pastel.orange }) => {
+export const BreathingAnimation: FC<Props> = ({ animationValue, color }) => {
   const { colorScheme } = useColorScheme();
+  const { customRotatingCircleColor } = useSettingsStore();
```

Comment:
Can we do this in https://github.com/mmazzarolo/breathly-app/blob/master/src/screens/exercise-screen/exercise-screen.tsx and pass the color down as a parameter to this component instead?